### PR TITLE
MuMuGamma_mll_0to60_LO for H->llg

### PR DESCRIPTION
**Pull request by hrjheng**
Pull request for MuMuGamma_mll_0to60_LO cards/sample for the Higgs Dalitz analysis

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuGamma_mll_0to60_LO/MuMuGamma_mll_0to60_LO_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuGamma_mll_0to60_LO/MuMuGamma_mll_0to60_LO_run_card.dat:66]: Why is matching turned ON? You don't have additional partons to your Born process

**Comment by sihyunjeon** [comment]
How are the MuMuJets card different from the old merge request #2855 ?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuGamma_mll_0to60_LO/MuMuGamma_mll_0to60_LO_run_card.dat (hunk 1 above)
Comment (hrjheng) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuGamma_mll_0to60_LO/MuMuGamma_mll_0to60_LO_run_card.dat]: Now the matching is turned off

**Comment by hrjheng** [comment]
The MuMuJets cards are the same as those in the old PR.

**Comment by andreypz** [comment]
I think once the #2855 is merged, this PR will only contain the MuMuGamma card.

**Comment by sihyunjeon** [comment]
Please remove the mumujet in here as it's merged

**Comment by sihyunjeon** [comment]
@agrohsje I am fine with MuMuGamma here

**Comment by hrjheng** [comment]
Now the files that contain changes are only MuMuGamma cards. Please let me know if there is anything I should fix. Thanks.

**Pull request merged**